x-pack/test/security_solution_endpoint_api_int/apis/resolver/entity_id.ts

```diff
@@ -147,11 +162,32 @@ export default function ({ getService }: FtrProviderContext) {
       });
 
       it('does not query for ancestors that have an empty string for the entity_id', async () => {
-        const { body }: { body: SafeResolverTree } = await supertest
-          .get(`/api/endpoint/resolver/${origin.process?.entity_id}`)
+        const hasID = (nodes: ResolverNode[], id: string | undefined): boolean => {
+          if (!id) {
+            return false;
+          }
+          return nodes.find((node) => node.id === id) !== undefined;
+        };
+        const { body }: { body: ResolverNode[] } = await supertest
+          .post('/api/endpoint/resolver/tree')
+          .set('kbn-xsrf', 'xxx')
+          .send({
+            descendants: 0,
+            ancestors: 10,
+            schema: schemaWithAncestry,
+            nodes: [origin.process?.entity_id],
+            timeRange: {
+              from: timestampAsDateSafeVersion(ancestor2)?.toISOString(),
+              to: timestampAsDateSafeVersion(origin)?.toISOString(),
+            },
+            indexPatterns: ['logs-*'],
+          })
           .expect(200);
-        expect(body.ancestry.ancestors.length).to.be(1);
```

Comment:
The reason for the difference in `1` and `2` here is because the older deprecated `/tree` API would place the origin node in its own field, and then the rest of the ancestors in the `ancestry` field. The new `/tree` API places everything in the same response array so it will have both the origin and the single valid ancestor.